src/sass/_theme_breadcrumbs.sass

```
@@ -1,21 +1,19 @@
 .wy-breadcrumbs
   +clearfix
 
-.wy-breadcrumbs li
+.wy-breadcrumbs > li
   display: inline-block
+  padding-top: 5px
   &.wy-breadcrumbs-aside
     float: right
-  a
-    display: inline-block
-    padding: 5px
-    &:first-child
-      padding-left: 0
+  /* Style literals as links */
   code
-    padding: 5px
-    border: none
-    background: none
-    &.literal
-      color: $text-color
+    all: inherit
+    color: inherit
+
+.breadcrumb-item::before
+  content: "/ "
```

The look of the new breadcrumb separators resulted a bit too intrusive.
I would maybe shrink them and make them lighter.

```suggestion
  content: "/"
  color: #bbb
  font-size: 13px
  padding: 0 6px 0 3px
```
_The new padding is required to re-center them, after changing the font size._

---

Hola cómo estás